packages/math/typesetter.lua

```diff
@@ -147,8 +147,8 @@ function ConvertMathML (_, content)
       if #children ~= 2 then
          SU.error("Wrong number of children in mfrac: " .. #children)
       end
-      return SU.boolean(content.options.bevelled, false)
-            and b.bevelledFraction(content.options, children[1], children[2])
+      return SU.boolean(content.options.beveled, false)
```

Comment:
So **NOT** here in any case. The [MathML standard](https://www.w3.org/TR/mathml4/#presm_mfrac_att) has the attribute with UK spelling: 
![image](https://github.com/user-attachments/assets/e6c1eb7e-8b11-4225-bd81-83c3f3aa50e6)

Reply:
(No strong opinion or objection for our own code and documentation - Just the attribute name must be the one from the W3C standard)

Reply:
Okay, thanks for the reference. I'll add an exception to `typos` instead of changing the spelling. As much as it looks wrong to me all of our references are going to be in regard to MathML, and if that's the spelling used in the standard I don't see any sense in mixing and matching.